Formula/imagemagick@6.rb

```diff
@@ -4,8 +4,8 @@ class ImagemagickAT6 < Formula
   # Please always use a mirror as the primary URL as the
   # ImageMagick site removes tarballs regularly which means we get issues
   # unnecessarily and older versions of the formula are broken.
```

Comment:
Does this comment still make sense?

Reply:
Probably not, no. We can remove it in a syntax-only PR after this one is merged, to avoid the extra CI run and merge conflict.